### PR TITLE
[BAAS-915] Update Card Presentment SDK

### PR DESCRIPTION
**Pull request by osylko-bondtech**
https://bond-financial.atlassian.net/browse/BAAS-915

### CODE REVIEW

File: docs/gen/BondCards.html [hunk 1]
```diff
@@ -261,7 +261,7 @@ <h3 class="subsection-title">Methods</h3>
             
 <hr>
 <dt>
-    <h4 class="name" id="field"><span class="type-signature"></span>field(selector, type [, compareValue] [, css] [, placeholder] [, successColor] [, errorColor] [, color] [, lineHeight] [, fontSize] [, fontFamily] [, disabled] [, readOnly] [, autoFocus])</h4>
+    <h4 class="name" id="field"><span class="type-signature"></span>field(selector, type [, css] [, placeholder] [, successColor] [, errorColor] [, color] [, lineHeight] [, fontSize] [, fontFamily] [, disabled] [, readOnly] [, autoFocus])</h4>
```
Comment (dominicsardina) [inline docs/gen/BondCards.html:264]: Maybe you didn't pull the latest before generating docs? It looks like it removed my compareValue additions. You just might need to make sure you merged master and rerun the doc generation.

File: src/bond-sdk-cards.js [hunk 2]
```diff
@@ -105,6 +105,103 @@ class BondCards {
...
+   * @property {String} format.withThis The string that replaces the substring
+   * specified by the specified regexp. ex: '$1-$2-$3-$4'
+   * @property {String} [format.count] Optional, defines how many times a certain
+   * string should be replaced.
+   * @property {Object} [css={}] An object of CSS rules to apply to the response.
+   */
+
+  /**
+   * The FieldType
+   * @typedef {('number'|'cvv'|'expiry')} FieldType
```
Comment (dominicsardina) [inline src/bond-sdk-cards.js:130]: I like using this fieldType. Can you update the jsDoc code for .show to match this?

**Review by dominicsardina — CHANGES_REQUESTED** [review]
1. When we add a new feature, we need to bump the version number in package.json (1.3.0 to 1.4.0 this time). That way when we publish to npm, there's a new version of the npm package for them to download.
2. I thought this multiple functionality with VGS was new from them, so wouldn't we need to upload a new show.js file that they offer? I don't see it.
3. What happens when one of the calls fail... everything retries? Error message? All three fields don't show, right?

File: src/bond-sdk-cards.js (hunk 2 above)
Comment (osylko-bondtech) [reply src/bond-sdk-cards.js:130]: Sure

**Comment by dominicsardina** [comment]
2. Yes, they released show.js v1.3 which has the capability to do multiple calls at once. That's why we went ahead with this ticket now. 

Here's the new show.js
https://js.verygoodvault.com/vgs-show/1.3/ACogZ2LNgMLr6gZNZiduenkv.js

We'll need to copy that code into the sdk, replacing the old file.

I don't see any documentation on the new feature tho. I asked @prithvi-bond in the #sdk slack channel if he has seen it.

**Review by dominicsardina — APPROVED** [review]

**Pull request merged**